src/cpp/fastdds/subscriber/SubscriberImpl.cpp

```diff
@@ -471,9 +471,13 @@ void SubscriberImpl::SubscriberReaderListener::on_liveliness_changed(
         DataReader* reader,
         const fastrtps::LivelinessChangedStatus& status)
 {
+    (void)status;
+
     if (subscriber_->listener_ != nullptr)
```

Comment:
If we are now checking the status mask on the DataReader, I'd add the check here too.